doc/conf.py

```diff
@@ -43,6 +43,11 @@
 html_theme = "pydata_sphinx_theme"
 html_favicon = "_static/icons/favicon.ico"
 
+current_release = panel.__version__  # Current release version variable
+
+announcement_text = f"Panel {current_release} has just been released! Check out the <a href='https://panel.holoviz.org/about/releases.html#version-1-4-0'>release notes</a> and support Panel by giving it a 🌟 on <a href='https://github.com/holoviz/panel'>Github</a>."
```

Comment:
Arf unfortunately this is not quite correct, `annoucement_text` contains a link to `releases.html#version-1-4-0` ... @Coderambling can you please open a new PR to remove this fragment? No need to build it dynamically I think, since we always want users to see the top of the release notes page.

Reply:
Yes wel spotted, I will replace that link with the one below ok? That will show the top of the release page.

https://panel.holoviz.org/about/releases.html

Reply:
Ok.

Reply:
Also, as the next PR after that, the Github API could be used to retrieve the release date,  so instead the text would then be

Panel `current_release` was released on `release_date`.

Would that be an improvement?

Example code to retrieve release date:


`import requests


url = "https://api.github.com/repos/holoviz/panel/releases/latest"

response = requests.get(url)
data = response.json()

print("The latest release date is:", data['published_at'])
`

It would mean importing the requests library in conf.py though. Would that be an issue?

Advantage is that it could then also be used to retrieve the number of Github Panel downloads and forks for display on the site

This would be a good way to communicate to (first time) site visitors that Panel is in active development and is a popular framework.

Reply:
Updated the hyperlink:

https://github.com/holoviz/panel/pull/7046